data/core/units/monsters/Shadow_Jumping_Spider.cfg

```diff
@@ -26,9 +26,9 @@
     cost=58
     usage=fighter
     undead_variation=spider
-    description= _ "Unlike the feared Giant Spider, that uses poison and webs to catch prey, the Shadow Jumper relies on speed and the element of surprise.  These stealthy, keen-eyed hunters creep to within striking distance, then pounce with a piercing bite that is said to punch through steel.  Once they have pierced their victim's defenses, the spiders take what nourishment is available, feeding in a manner as fast and remorseless as befits their nature.
+    description= _ "Unlike the feared Giant Spider that uses poison and webs to catch prey, the Shadow Jumper relies on speed and the element of surprise.  These stealthy, keen-eyed hunters creep to within striking distance, then pounce with a piercing bite that is said to punch through steel.  Once they have pierced their victim's defenses, the spiders take what nourishment is available, feeding in a manner as fast and remorseless as befits their nature.
```

Comment:
I'd take out the first `piercing`. The bite punches through steel, and `pierced` appears in the next sentence.

Reply:
That's a bit more in the domain of modifying the text, which I was trying to avoid given all the blockers on string modification. But since nemaara has already requested some changes I may as well follow through with this one too.